src/Resources/config/routing.yml

```diff
@@ -1,30 +1,19 @@
 simple_things_entity_audit_home:
-    path: /{page}
+    path: /{page<\d+>?1}
```

Comment:
IMO, these changes should be reverted.

Reply:
We do not have to but I should add next_maroj note about remove this file.

Reply:
Reverted. Also I add NEXT_MAJOR to remove this file.